examples/manifests/consumer.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           args:
             - "--local-api-addr=127.0.0.1:9089"
             - "--api-path=/api/ocloudNotifications/v1/"
-            - "--api-addr=127.0.0.1:8089"
```

Comment:
why change the port?

Reply:
to be consistent with documentation and the port used at ptp/bmer operators.